### PR TITLE
add string.split function for empty string delimiters

### PR DESCRIPTION
**Pull request by vnksnkr**
PR for #280 
added functionality to split string into individual characters if empty string is passed as delimiters

### CODE REVIEW

File: c/datatypes/strings.c [hunk 1]
```diff
@@ -135,7 +135,20 @@ static Value splitString(VM *vm, int argCount, Value *args) {
 
     ObjList *list = initList(vm);
     push(vm, OBJ_VAL(list));
-
+    if(!strcmp(AS_CSTRING(args[1]),""))
```
[the remark this answers is not in the document]
Comment (vnksnkr) [reply c/datatypes/strings.c:138]: Thanks for pointing it out..will make a new PR with the mentioned changes

[the remark this answers is not in the document]
Comment (Jason2605) [reply c/datatypes/strings.c:138]: You don’t need to make a new PR you can just push any changes to this one!

[the remark this answers is not in the document]
Comment (vnksnkr) [reply c/datatypes/strings.c]: Have made the changes

[the remark this answers is not in the document]
Comment (Jason2605) [reply c/datatypes/strings.c]: I pushed a slight update to the branch for a few formatting changes. I think the only thing left for the PR is to add a test entry for this and a note in the `str.split` documentation about this new functionality

[the remark this answers is not in the document]
Comment (vnksnkr) [reply c/datatypes/strings.c]: @Jason2605 could you clarify where the test entry is to be added? Is it in the .du files ?Apologies if I didnt get your suggestion

[the remark this answers is not in the document]
Comment (Jason2605) [reply c/datatypes/strings.c]: Of course!

A new assertion in this test file (https://github.com/dictu-lang/Dictu/blob/develop/tests/strings/split.du)
A note about the new functionality in this documentation file (https://github.com/dictu-lang/Dictu/blob/develop/docs/docs/strings.md)

**Review by Jason2605 — APPROVED** [review]

**Comment by Jason2605** [comment]
Perfect, thank you very much!

**Pull request merged**